pkg/clients/integration/pipelineruns.go

```diff
@@ -180,7 +180,7 @@ func (i *IntegrationController) WaitForAllIntegrationPipelinesToBeFinished(testN
 	return nil
 }
 
-// WaitForBuildPipelineRunToBeSigned waits for given build pipeline to get signed.
+// WaitForBuildPipelineRunToBeSigned waits for given build pipeline to get signed annotation.
 // In case of failure, this function retries till it gets timed out.
 func (i *IntegrationController) WaitForBuildPipelineRunToBeSigned(testNamespace, applicationName, componentName string) error {
```

Comment:
I'm thinking that maybe we should remove this function as whatever it's trying to do now can be done by a more generic function here: https://github.com/redhat-appstudio/e2e-tests/blob/f6c112bb246795194181806f0383611a6fcd60a5/pkg/clients/integration/pipelineruns.go#L211

Reply:
agree, 
wdyt @MartinBasti ?

Reply:
works for me

Reply:
Fixed

Reply:
Thanks!